app.js

```diff
@@ -67,32 +70,52 @@ for (var i = 0; i < stores.length; i++) {
   rowEl.appendChild(dailyTotalEl);
 }
 
+for (var i = 0; i < storeHours.length; i++) {
```

Comment:
Break this out into a function! That way you'll be able to calculate the totals whenever you want.